Core/Core/Database/CoreData/Models/Course.swift

```diff
@@ -68,6 +68,45 @@ extension Course {
         let color = try? managedObjectContext?.fetch(request).first
         return color??.color ?? .named(.ash)
     }
+
+    static let scoreFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .percent
+        formatter.decimalSeparator = "."
+        formatter.multiplier = 1
+        formatter.maximumFractionDigits = 3
+        formatter.roundingMode = .down
+        return formatter
+    }()
+
+    public var widgetDisplayGrade: String {
```

Comment:
Do you think we can call this `displayGrade`? Ideally the widget will show the same thing as the course cards and everywhere else, right?